javascript/package.json

```diff
@@ -16,13 +16,13 @@
   "devDependencies": {
     "@tsconfig/node16": "1.0.2",
     "@types/node": "16.10.3",
-    "protoc-gen-ts": "0.6.0",
-    "typescript": "4.4.3"
+    "protoc-gen-ts": "0.8.6",
```

Comment:
Did you confirm that this version is ok since 0.8.5 was a breaking change?

Reply:
The easiest way to confirm that is going to be to release this artifact and then try to upgrade the JS SDKs to consume it.  If it doesn't work then they will skip this version, and we will revert this if necessary.